cmd/helm/root.go

```diff
@@ -100,7 +101,9 @@ Common actions from this point include:
 - helm list:      list releases of charts
 
 Environment:
-  $HELM_HOME          set an alternative location for Helm files. By default, these are stored in ~/.helm
+  $XDG_CACHE_HOME     set an alternative location for storing cached files.
```

Comment:
These variables should use the XDG prefix as changing this variable affects the base directory. It doesn't change the `helm` directory itself, so this is correct.

As an example, by default on linux, this would be `~/.config/helm`. By changing $XDG_CACHE_HOME to `/tmp`, the alternative location is now `/tmp/helm`.

For example, on my Ubuntu machine, the Operating System sets $XDG_CACHE_HOME to `/home/bacongobbler/snap/code/common/.cache/` by default. So, the expected output of `helm path cache` is

```
><> ./bin/helm path cache
/home/bacongobbler/snap/code/common/.cache/helm
```

Does that make sense?

Reply:
Yep, that makes sense